### DISCUSSION TITLE
Latin Vulgate downloading fails at the last file.

### DISCUSSION THREAD

**Discussion by HubKing** [crosswire/xiphos#1110] [Q&A]
I have tried multiple times/restarting but it keeps failing at the 6th file. 

Please ignore the cursor; I think the capture programme has a bug for Wayland.

https://user-images.githubusercontent.com/10900489/164464737-b80ecce3-2c17-459b-a59c-1d2365b536eb.mp4

**Comment by karlkleinpaste** [comment]
I just re-downloaded Vulgate myself with no problem at all. I did several other installations and updates, again without problem. Thus the problem must be at your end.

A random guess is that you have a permissions problem in the area under your `.sword` directory where the modules reside. Ensure that you do not lack write permissions on each of the subdirectories there. For Vulgate, this will be under `.sword/modules/texts/ztext/vulgate`.

FYI the installer subsystem operates by downloading content into a temporary area under `.sword/InstallMgr` and then moving the content to its proper destination.

**Reply by HubKing** [reply]
Well, I tested it on another system (Endeavour OS) and it worked there, but not this computer (Manjaro) even after deleting the `.sword` and `.xiphos` directories. Is there any log or something that I can see the details of why it failed, instead of just the "failed" message?

PS: Only Vulgate fails. Other versions are installed fine. I have also tried running it as root (sudo xiphos) and installing Vulgate still failed in the exact same way.

**Comment by LAfricain** [comment]
Be patient and try an other day. I have the same issue some time randomly. Probably an internet issue.
